manga_ocr_dev/training/get_model.py

```diff
@@ -20,7 +19,7 @@ def __init__(self, feature_extractor, tokenizer):
 
 
 def get_processor(encoder_name, decoder_name):
-    feature_extractor = AutoFeatureExtractor.from_pretrained(encoder_name)
+    feature_extractor = AutoImageProcessor.from_pretrained(encoder_name, use_fast=True)
```

Comment:
AutoFeatureExtractor is deprecating, as in #40 , and `use_fast=True` is suggested.